### DISCUSSION TITLE
Does this not work when Andorid's VPN is enabled?

### DISCUSSION THREAD

**Discussion by amithm7** [subins2000/WebDrop#16] [General]
I'm not using a VPN, but a [DoH / Firewall app](https://rethinkfirewall.com), that works through Android's VPN interface.

WebDrop is unable to detect devices on the same network / WiFi until I turn off the VPN.

**Comment by subins2000** [comment]
Check if the room code is the same. You can find the room code by clicking the "Share invite link" button (the globe icon).

**Reply by amithm7** [reply]
Yes, the room code is the same.

Also, I just noticed. It works in Brave Browser (Chrome 90) for some reason. On both Firefox 88 and Chrome 90, it is only able to detect other browsers in the same device; not the one on PC.

I'm on Android 11.

**Reply by subins2000** [reply]
Device can only be detected if WebRTC connection can be established. Maybe when the Android's VPN is enabled WebRTC fail to connect in those browsers. I've had this issue in Chrome Android webview once.
